Detections/MultipleDataSources/WSLMalwareCorrelation.yaml

```diff
@@ -0,0 +1,148 @@
+id: d992b87b-eb49-4a9d-aa96-baacf9d26247
+name: Alert for  IOCs related to Windows/ELF malware - IP, Hash IOCs - September 2021
+description: | 
+  'Identifies a match across various data feeds for IP,hashes and IOCs related to Windows/ELF malware published by Black Lotus Labs'
+severity: Medium
+requiredDataConnectors: 
+  - connectorId: F5
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: CiscoASA
+    dataTypes: 
+      - CommonSecurityLog
+  - connectorId: PaloAltoNetworks
+    dataTypes: 
+      - CommonSecurityLog
+  - connectorId: Fortinet
+    dataTypes: 
+      - CommonSecurityLog
+  - connectorId: CheckPoint
+    dataTypes: 
+      - CommonSecurityLog
+  - connectorId: CEF
+    dataTypes: 
+      - CommonSecurityLog               
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - DeviceNetworkEvents
+      - DeviceFileEvents
+      - DeviceEvents
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+  - connectorId: AzureFirewall
+    dataTypes: 
+      - AzureDiagnostics
+  - connectorId: WindowsFirewall
+    dataTypes:
+      - WindowsFirewall   
+queryFrequency: 6h 
+queryPeriod: 6h 
+triggerOperator: gt 
+triggerThreshold: 0 
+tactics: 
+  - Impact
+relevantTechniques:
+  - T1496
+tags:
+  - ELF malware
+query:  |  
+    let IPList = dynamic(["185.63.90.137"]);  
+    let IPRegex = '[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}';
+    let sha256Hashes = 
+    dynamic(["53854c6d163bfd0c56d8b297ac43bd25c21f696de6063031241e792ee65df441",
+    "c297e545b8f150cc5ff56dbb68dc74fe30a421d9d40f38f4a53083192697c44c",
+    "17921368901f23e0cad0d2fe4ce5694aebaf4727699ed0358117500701914d1b",
+    "198a2d42df010d838b4207f478d885ef36e3db13b1744d673e221b828c28bf77",
+    "71d7b48c2fdc7b57b104a7858a35165bbed21d2fa7e34828d6c1d50b2b33a1d0",
+    "601227d52c6e367e11b80240183d07d38bc11a88e844e8401fce17eb25e92ba8",
+    "63ff04bed4fdb120a9cb9b1ea7fd88e83f12fb01ab6a057088f8016e663b48d4",
+    "a3037c3389b811bc1404f719af5c8b9034c5e24710cf3a0b457d28bf1b922cf7",
+    "e19b8be1b21c066d60725e550f8455f824065abbf1b43f7b2fe4fb338b241ffc",
+    "a3037c3389b811bc1404f719af5c8b9034c5e24710cf3a0b457d28bf1b922cf7"
+    ]);
+    (union isfuzzy=true
+    (CommonSecurityLog
+    | where SourceIP in (IPList) or DestinationIP in (IPList) or Message has_any (IPList) 
+    | parse Message with * '(' DNSName ')' * 
+    | project TimeGenerated, SourceIP, DestinationIP, Message, SourceUserID, RequestURL
+    | extend MessageIP = extract(IPRegex, 0, Message)
+    | extend IPMatch = case(SourceIP in (IPList), "SourceIP", DestinationIP in (IPList), "DestinationIP", MessageIP in (IPList), "Message", MessageIP in (IPList), "Message", "NoMatch"),  AlertDetail = 'Alert for  IOCs related to Windows/ELF malware'
```

Comment:
I don't think we need the 'AlertDetail = 'Alert for IOCs related to Windows/ELF malware' part in the query.  The description field of the query is already populated for the customer to understand what the detection query is trying to do. Can we please remove it from all the different data sources where we have in the query. Thanks.

Reply:
Agreed, not needed.